### DISCUSSION TITLE
plugins/value-added

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#270] [文档评论区]
# plugins/value-added

Zotero 非官方中文维护小组

https://zotero-chinese.com/plugins/value-added

**Comment by whubao** [comment]
感谢插件商店帮我们上架这么多免费的插件。
插件商店能否组织开发者开发特殊插件，如集成scispace， semantic scholar， researchrabbit，connected paper 等引用关系或可视化的插件？
